spec/models/reviewable_spec.rb

```diff
@@ -244,6 +244,55 @@
     end
   end
 
+  context ".recent_list_with_pending_first" do
+    fab!(:pending_reviewable1) { Fabricate(:reviewable, score: 150) }
+    fab!(:pending_reviewable2) { Fabricate(:reviewable, score: 100) }
+    fab!(:approved_reviewable1) { Fabricate(:reviewable, score: 300) }
+    fab!(:approved_reviewable2) { Fabricate(:reviewable, score: 200) }
+
+    fab!(:admin) { Fabricate(:admin) }
+
+    it "returns a list of reviewables with pending items first" do
+      list = Reviewable.recent_list_with_pending_first(admin)
+      expect(list).to contain_exactly(
+        pending_reviewable1,
+        pending_reviewable2,
+        approved_reviewable1,
+        approved_reviewable2,
+      )
```

Comment:
I think this assertion doesn't really fit into the description of what we're testing here. I think we can just leave this out.

Reply:
Not sure what I was thinking here, I've re-written this spec let me know what you think.